dev/z.ts

```diff
@@ -0,0 +1,26 @@
+// https://github.com/rupa/z
+const completionSpec: Fig.Spec = {
+  name: "z",
+  args: {
+    name: "regex",
+    isVariadic: true,
```

Comment:
Seems like the regex arg is also optional `z [-chlrtx] [regex1 regex2 ... regexn]`

```suggestion
    isVariadic: true,
    isOptional: true,
```

Reply:
I am not sure about this. It may just be a syntax misaligned.
@fwesss does it anything if you don't specify args?

Reply:
Running `z` with no args outputs the history. Equivalent to running `z -l`.